config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presubmits.yaml

```diff
@@ -32,14 +32,28 @@ presets:
       secretKeyRef:
         name: clusterapi-provider-vsphere-ci-prow
         key: target-vm-ssh-pub
+  - name: VM_SSH_PUB_KEY
+    valueFrom:
+      secretKeyRef:
+        name: clusterapi-provider-vsphere-ci-prow
+        key: vm-ssh-pub-key
   volumeMounts:
   - name: jumphost-key
     mountPath: /root/ssh/.jumphost
+  - name: private-key
+    mountPath: /root/ssh/.private-key
   - name: bootstrapper-key
     mountPath: /root/ssh/.bootstrapper
   - name: vpn-conf
     mountPath: /root/.openvpn
   volumes:
+  - name: ssh-key
+    secret:
+      secretName: clusterapi-provider-vsphere-ci-prow
+      defaultMode: 256
+      items:
+      - key: vm-ssh-key
```

Comment:
as @gab-satchi pointed this needs to be an env variable, to be able to consume it in your PR